cloudify_hostpool/tests/config/test_config.py

```diff
@@ -0,0 +1,130 @@
+# #######
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+import os
+import StringIO
+import socket
+import unittest
+
+from mock import MagicMock
+from mock import patch
+
+from cloudify_hostpool.config import config
+
+
+class ConfigurationTest(unittest.TestCase):
+    def test_get_broadcast(self):
+        ip = '123.213.123.232/30'
+        self.assertEqual(('123.213.123.232', '30'),
+                         config._get_subnet_and_mask(ip))
+
+    def test_get_subnet_and_mask(self):
+        ip = '123.213.123.232/30'
+        self.assertEqual(('123.213.123.232', '30'),
+                         config._get_subnet_and_mask(ip))
```

Comment:
there are two identical tests here...
test_get_broadcast and test_get_subnet_and_mask